docs/DeveloperGuide.md

```diff
@@ -2,23 +2,50 @@
 layout: page
 title: Developer Guide
 ---
- Table of Contents:
- * [Setting up, getting started](#setting-up-getting-started)
- * [Design](#design)
- * [Implementation](#implementation)
- * [Appendix: Requirements](#appendix-requirements)
- * [Appendix: Instructions for manual testing](#appendix-instructions-for-manual-testing)
---------------------------------------------------------------------------------------------------------------------
-
-## **Setting up, getting started**
+# GrAB3 Developer Guide
+
+## Table of Contents:
+ * [1. Introduction](#1-introduction)
+ * [2. Setting up, getting started](#2-setting-up-getting-started)
+ * [3. Design](#3-design)
+ * [4. Implementation](#4-implementation)
+ * [5. Documentation](#5-documentation)
+ * [6. Logging](#6-logging)
+ * [7. Testing](#7-testing)
+ * [8. Configuration](#8-configuration)
+ * [9. DevOps](#9-dev-ops)
+ * [Appendix A: Product Scope](#appendix-a-product-scope)
+ * [Appendix B: User Stories](#appendix-b-user-stories)
+ * [Appendix C: Use Cases](#appendix-c-use-cases)
+ * [Appendix D: Non Functional Requirements](#appendix-d-non-functional-requirements)
+ * [Appendix E: Glossary](#appendix-e-glossary)
+ * [Appendix F: Instructions for Manual Testing](#appendix-f-instructions-for-manual-testing)
+ * [Appendix G: Effort](#appendix-g-effort)
+
+## 1. **Introduction**
+
+**GrAB3** is a desktop business management application meant for home-based beauty salon owners who want to consolidate all
+the information related to their business, such as upcoming appointments, their list of clients, as well as profits recorded. 
+It focuses on the Command Line Interface (CLI) while providing users with a simple and clean Graphical User Interface (GUI).
+Thus, the main interaction with **GrAB3**. will be done through commands.
+
+The features of GrAB3 include:
+* Keeping track of existing/new clients 
+* Scheduling of upcoming appointments with clients
+* Recording of revenue from services provided
+* Recording of business-related expenses
+* Calculation of business profit.
```

Comment:
Think you might have added an extra '.' at the end!

Reply:
Resolved!